### ISSUE TITLE
Add survey weights as variable within `extract_dhs`

### ISSUE THREAD

**Issue by OJWatson** [OJWatson/rdhs#52]
It makes sense to always extract survey weights when extracting, so this should be another argument to `extract_dhs(add_weights = TRUE)`.

**Comment by jeffeaton** [comment]
If doing that, it probably also makes sense to also always extract the clusters.

**Comment by OJWatson** [comment]
Yup agreed - in fact will just have one argument that adds the main useful consistent covariates (cluster, weight, admin (already in the `add_geo` option)).

Any others?
________________________________

**Comment by OJWatson** [comment]
For example, the following is annoying. Have an argument that adds common useful variables, (cluster, weights) and maybe even does the 1e6 correction:

```
qus <- search_variable_labels(datasets, "Received POLIO 3|cluster|sample weight|region")
extr <- extract_dhs(qus)

data <- rbind_labelled(extr, labels = list("h8"="concatenate"))
data$weight <- data$v005 / 1e6
```
